src/MatBlazor/wwwroot/dist/matBlazor.css

```diff
@@ -1,3 +1,3 @@
 @import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500);
```

Comment:
Not for this PR, but eventually, we need to move the .css files to a readable format, and use build steps to minify these on output.

Reply:
That would be nice. Editing minified css is a pain.